acsylla/_cython/statement/statement.pyx

```diff
@@ -39,20 +42,23 @@ cdef class Statement:
             object page_size,
             object page_state,
             object timeout,
-            object consistency):
+            object consistency,
+            object serial_consistency):
 
         cdef Statement statement
 
         statement = Statement()
         statement.cass_statement = cass_statement
         statement.cass_prepared = cass_prepared
         statement.prepared = 1
-        statement._set_paging(page_size, page_state)
-        statement._set_timeout(timeout)
-        statement._set_consistency(consistency)
+        statement.set_page_size(page_size)
+        statement.set_page_state(page_state)
+        statement.set_timeout(timeout)
+        statement.set_consistency(consistency)
+        statement.set_serial_consistency(serial_consistency)
         return statement
 
-    cdef _set_paging(self, object py_page_size, object py_page_state):
+    cpdef set_page_size(self, object py_page_size):
```

Comment:
Seems that with this change we are exposing two ways of setting the page size and the binding, Im wondering if we should try to prescribe just one way instead of supporting two? Originally the state and the size was configured during the `bind`.

What are your thoughts on that?

Reply:
I think we should be able to change the settings for statement on the fly because statement can be reused

Reply:
Then maybe the `page_state` should be only provided explicitly by the new method and remove it from the `bind` method? just for the sake of having a unique way of doing things, otherwise the user might get confused.

What are other implementations doing?

In any case we should cover this using a test that checks that this methods are working as expected, either modified the current that we have for paging or by creating a new one.

Reply:
As I see, for example, the `python-driver` allows set this params directly
https://github.com/scylladb/python-driver/blob/master/cassandra/query.py#L213

Reply:
I will add more tests

Reply:
lets then keep on interface for the state, mabye the one that you provided is the one that makes more sense since the user does not need to bind again the statement?

Reply:
yeah in this implementation we may reuse statement,  example for pagging:

```python
    statement = prepared.bind(page_size=10)
    while True:
        result = await session.execute(statement)
        if result.has_more_pages():
            statement.set_page_size(100)
            statement.set_page_state(result.page_state())
        else:
            break
```
and other cases